source/getting_started.rst

```diff
@@ -4,82 +4,303 @@
+        ::
+
+            wget https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
+
+2. Install Conda
+
+    Linux
+        ::
+
+            bash ./Miniconda3-latest-Linux-x86_64.sh
+
+
+    MacOS
+        ::
+
+            bash ./Miniconda3-latest-MacOSX-x86_64.sh
+
+    - ``Do you wish the installer to initialize Miniconda3 by running conda init? [yes|no] yes``
+
+3. If you are working on a machine/network that intercepts SSL communications (such as
+acme1), you will get an SSL error unless you disable the SSL verification:
+
+    ::
+
+        conda config --set ssl_verify false
+        binstar config --set ssl_verify False
+
+4. Set the following:
+
+    ::
+
+        conda config --add channels conda-forge
+        conda config --set channel_priority strict
+
+5. Once conda is properly working, you can install the **(a) Latest Stable Release** or
+create a **(b) Development Environment**.
+
+(a) Latest Stable Release
+=========================
+
+Installation using conda
+------------------------
+
+First, make sure that you're using ``bash``. ::
 
-As a first example, let's say we want to post-process 100 years
-of an existing simulation. In particular, we'd like to generate
-the following:
+   $ bash
 
-* Regridded atmosphere climatologies every 20 and 50 years
-* Regridded atmosphere monthly time series files in 10 year chunks
-* E3SM Diags every 20 and 50 years
-* MPAS-Analysis
+You must have Anaconda installed as well.
+See :ref:`"Installation in a Conda Environment" <conda_environment>` section above for
+installing conda.
+Create a new Anaconda environment with ``zppy`` installed and activate it: ::
 
-First, create a configuration file in your favorite editor: ::
+   $ conda create -n zppy_env -c e3sm -c conda-forge zppy
+   $ source activate zppy_env
 
-  cd E3SM/script
-  gvim post.mysimulation.cfg
+Or you can install ``zppy`` in an existing environment. ::
 
-Copy and paste the following:
+   $ conda install zppy -c e3sm -c conda-forge
 
-.. literalinclude:: post.mysimulation.cfg
-   :language: cfg
-   :linenos:
+Updating
+--------
 
-The configuration files consists of sections ([...]) and subsections ([[...]]). There is
-a default section at the top ([default]) to define some common settings, followed
-by a separate section for each available tasks. Within each task section, you can optionally
+If you **installed via Anaconda** (e.g., not through the unified environment),
+you can update ``zppy`` by doing the following:  ::
+
+    conda update zppy -c e3sm -c conda-forge
+
+.. _dev-env:
+
+(b) Development Environment
+===========================
+
+Unlike the latest stable release (i.e., the user environment), the development
+environment does not include ``zppy``.
+Instead, the developer will ``python -m pip install .`` to build ``zppy`` with changes
+(see step 6 below).
+
+..
+    # TODO: Add the QA tools
```

Comment:
I copied the installation steps from  the `zstash` docs, so I figured I would just comment out (`..` in `rst` files) text about the QA and pre-commit tools for now.